src/html_support_files/odoc.css

```diff
@@ -1285,29 +1284,40 @@ td.def-doc *:first-child {
       "content"
       "toc-global";
     grid-template-columns: 1fr;
-  }
-
-
-  .odoc-search {
-    position: relative;
-    padding-bottom: 0;
-    height: calc(var(--search-bar-height) + var(--search-padding-top));
-  }
-
-  .odoc-tocs {
+    &:has(> .odoc-search:focus-within) {
+      /* This is the same as when there is no focus on the search bar, this is
+      just to prevent the full screen rule from changing anything. */
```

Comment:
What do you mean with "fullscreen rule"?

Reply:
I meant the large screen/default rule

Reply:
```suggestion
      just to prevent the "wide layout" rule from changing anything. */
```